pkg/codegen/nodejs/gen_program.go

```diff
@@ -179,7 +179,7 @@ func (g *generator) genPreamble(w io.Writer, program *hcl2.Program) {
 			continue
 		}
 		as := makeValidIdentifier(path.Base(pkg))
-		if as != pkg {
+		if as != pkg || pkg == "crypto" {
```

Comment:
Why is this needed?

Reply:
`import * from "crypto"` doesn't compile properly when the `ComputeSHA1` method is added, and so I force it to use `import * as crypto from "crypto"` instead.